src/powfacpy/base_interface.py

```diff
@@ -1003,8 +1003,27 @@ def get_name_with_ending(objects):
     if not type(objects) == list:
       objects = [objects]
     return [x.GetFullName().split('\\')[-1] for x in objects]
-    
 
-if __name__ == "__main__":
-  print("ok")
-  
\ No newline at end of file
+
+def set_attr_of_obj(obj, attributes:dict):
```

Comment:
for consistency, add object type:
`set_attribute_of_object(object:powerfactory.DataObject, attributes:dict):`

Reply:
The problem (here and in many other cases) is that powerfactory is not available and not imported because the path is different for every user.